src/xdp-utils.c

```diff
@@ -736,11 +736,13 @@ parse_status_field_pid (const char *val,
 {
   const char *t;
 
+  // This is a tab separated list of namespaces. In the case of no namespaces
+  // there will be one entry.
   t = strrchr (val, '\t');
-  if (t == NULL)
-    return -ENOENT;
+  if (t != NULL)
+    val = t;
 
-  return parse_pid (t, pid);
+  return parse_pid (val, pid);
```

Comment:
This will make `Pid` parsing this too, but I guess that is harmless.

Reply:
Yes that will work the same.

The key difference here seems to be that `g_strstrip()` gets called before on the `NSPid` path (so no leading `\t`).

On the `Pid` path it passes the whole line including whitespace.

Reply:
I split these into two functions. They want slightly different things.